services/common/src/redux/utils/Validate.ts

```diff
@@ -324,21 +312,6 @@ export const validateIncidentDate = memoize((reportedDate) => (value) =>
     : undefined
 );
 
-/**
- * @param data: options to choose from
- * @param allowEmptyData: perform validation with empty data array (free text on empty data set will be invalid)
- */
-export const validateSelectOptions = memoize((data: IOption[], allowEmptyData = false) => (value):
```

Comment:
Validate functions that look like this are not necessary anymore with new antd, and have a tendency to cause issues.
Also deleted the postcode function that depended on provinceOptions being passed in as a prop- it broke spectacularly.